engine/crates/engine-v2/codegen/domain/schema.graphql

```diff
@@ -113,12 +113,19 @@ type EnumValue @meta(module: "enum_value") @indexed(id_size: "u32", max_id: "MAX
   directives: [TypeSystemDirective!]!
 }
 
+type JoinMemberDefinition @meta(module: "join_member", derive: ["Clone"]) {
```

Comment:
for stuff that is obviously `Copy` you can use `@copy` which will add `Copy` to the struct and generate a walker that copies the struct rather than keep a ref.